utils/b2d.go

```diff
@@ -145,6 +145,10 @@ func (b *B2dUtils) DownloadLatestBoot2Docker() error {
 		return err
 	}
 
+	return b.DownloadBoot2Docker(latestReleaseUrl)
+}
+
+func (b *B2dUtils) DownloadBoot2Docker(latestReleaseUrl string) error {
```

Comment:
How about naming method "DownloadISOFromURL".  Also the more misleading part is the message below that says "Downloading latest boot2docker release to ".  I can change that to "Downloading %s to %s ", latestReleaseUrl, machineName